src/Core/src/Platform/ReflectionExtensions.cs

```diff
@@ -0,0 +1,95 @@
+﻿#nullable enable
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+
+namespace Microsoft.Maui
```

Comment:
Do we want this public ?

Reply:
Was internal in Xamarin.Forms, should we changed it?, any reason to not be public?